lxd/devices.go

```diff
@@ -195,7 +206,7 @@ func deviceTaskBalance(d *Daemon) {
 		conf := c.ExpandedConfig()
 		cpu, ok := conf["limits.cpu"]
 		if !ok || cpu == "" {
-			currentCPUs, err := deviceGetCurrentCPUs()
+			currentCPUs, err := cGroupGet("cpuset", "/lxc", "cpuset.effective_cpus")
```

Comment:
Can we do this once and re-use the value? During scale testings with thousands of containers, this will cause a ton of reads.